netsim/ansible/templates/initial/sros.j2

```diff
@@ -80,13 +78,18 @@
 {%  set v6 = l.ipv6|default(False) %}
 {%  set in_base = l.vrf is not defined and (l.vlan is not defined or l.vlan.mode|default('irb')=='route') %}
 
-{%  if l.name is defined %}
+{%  if l.type == "loopback" %}
+{%   set desc = "Loopback" %}
+{%  elif l.name is defined %}
 {%   set desc = l.name|replace('->','~') + " (" + l.role|default('') + ")" %}
-{%  elif l.type|default("") == "stub" %}
+{%  elif l.ifname == "system" %}
+{%   set desc = "System interface" %}
```

Comment:
Does this ever get reached? System is a loopback, so I think this check should come first